build.gradle

```diff
@@ -22,7 +22,19 @@ dependencies {
 
 mainClassName = 'com.gradle.exportapi.Application'
 
+task validateDbConfig {
+    description 'Ensure that the target database properties file exists'
+    group 'Verification'
+
+    ext.configuration = file('src/main/resources/POSTGRES.properties')
```

Comment:
Configuration is such an overloaded term in Gradle (configuration phase, configuration closures, org.gradle.api.artifacts.Configuration interface) that I try to use the term very sparingly.

I recommend calling this pgconfig or dbconfig or something else which disambiguates from all the other usages of "Configuration"